WooCommerce/Classes/POS/Presentation/Reusable Views/POSBottomShadowViewModifier.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+
+/// Applies a shadow to the bottom of the View component
+///
+struct POSBottomShadowViewModifier: ViewModifier {
+    func body(content: Content) -> some View {
+        content
+            .background(
+            Color.white
+                .shadow(color: Color(.secondarySystemFill), radius: 10, x: 0, y: 0)
+                .mask(Rectangle().padding(.bottom, -20))
```

Comment:
A piece of visual feedback. This shadow is not transparent and doesn't create a real shadow effect. When scrolling up, you can observe how cards are cut-off when in a header should drop a smooth shadow on top of cards:

On the left - design, on the right - the app.
<img width="1790" alt="Shadow is not transparent" src="https://github.com/user-attachments/assets/fcfa928f-78fc-42cc-a268-5b1f0b401b33">

Reply:
Good point, I attempted to apply `Color.clear` here, as well as `EmptyView`, and similar attempts to improve its transparency, but when I do I see the `.shadow` effects not being properly applied. Is there anywhere in the codebase (or resource) you'd recommend I can check further?

Reply:
Got it. Intuitively I would think that shadow needs to be applied not to any `EmptyView` or `Color.white` but to `self`.

However, I don't know from the top of my head. I would need to research & experiment myself to see how to get this result.